tests/test_vmware.py

```diff
@@ -544,3 +544,45 @@ def test_vmware_overlapping_with_another_sdata(
     record_property("message", message)
 
     assert result_count == 1
+
+#
+# <7>2023-09-29T12:01:00.124Z esxi01 auto-backup.sh.2904545.[2904558]: ConfigStore has been modified since the last backup
+# <7>2023-09-29T12:01:00.340Z esxi04 auto-backup.sh.2800924.[2800937]: ConfigStore has been modified since the last backup
+def test_linux_vmware_esx_autobackup(record_property, setup_wordlist, setup_splunk, setup_sc4s):
```

Comment:
We don't have `setup_wordlist` fixture, it was removed some time ago. Is it test really passing?

Reply:
Honestly IDK I've never gotten the dev/test env to work. I always assumed all these tests were looped through in the CI pipeline.

I just copied a previous test and updated the event to match my packet captures.

Reply:
Ok, I will return from vacation soon and on Wednesday / Thursday will fix it by myself. It seems that test will fail, because we haven't fixture `setup_wordlist `

Reply:
Have you had a chance to look into this yet?  
I'd like to get this test fixed so everything can be merged, but I just don't know enough about how the tests work to fix it.

Reply:
Sure, thanks for reminding, I will fix your test and will merge it

Reply:
Is this ready for merge now?

Reply:
@zyphermonkey your filter working incorrectly, with test data that you provided in test it's going to `nix:syslog`

Reply:
@zyphermonkey I prepare to fix it by yourself, you provide me test with 5 syntax error, that not running, and finally this filter that you provided don't working

Reply:
<img width="1443" alt="Screenshot 2023-10-27 at 18 53 19" src="https://github.com/splunk/splunk-connect-for-syslog/assets/138466237/1e0ee839-38c6-40af-a1a9-1bdd9f073381">

Reply:
@zyphermonkey is it parser really working? Because according to log format in test. It's not working. 
If you willl approve me that it's working I can delete this and merge this PR

Reply:
I made these changes to my local configs and they worked correctly. It looks like the only issue are my tests. 

I'm sorry about that, having never written tests for sc4s before I'm sure I thoroughly messed them up. :(

I'll delete the tests.

Reply:
Ok, if it working and you tested that's ok. It's not a problem that without test I will merge it today